<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SingleDMJForm</class>
 <widget class="QWidget" name="SingleDMJForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1">
     <item>
      <widget class="QGraphicsView" name="graphicsView_DMJ">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_left" stretch="0,0,0">
       <item>
        <widget class="QPushButton" name="pushButton_reduce">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>减少</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>增加</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="styleSheet">
          <string notr="true">border: 0px;
</string>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout" rowstretch="1,0,0,0,0,0">
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>温度</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_type">
            <property name="text">
             <string>类型</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit_type">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 25 9pt &quot;Microsoft YaHei&quot;;
color: rgb(255, 255, 255);</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>压力</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="lineEdit_D">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 25 9pt &quot;Microsoft YaHei&quot;;
color: rgb(255, 255, 255);</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>地址</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="lineEdit_T">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 25 9pt &quot;Microsoft YaHei&quot;;
color: rgb(255, 255, 255);</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="lineEdit_P">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 25 9pt &quot;Microsoft YaHei&quot;;
color: rgb(255, 255, 255);</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>密度</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEdit_address">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 25 9pt &quot;Microsoft YaHei&quot;;
color: rgb(255, 255, 255);</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBox_type">
            <item>
             <property name="text">
              <string>60PR</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>100PR</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>HM100PR</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>100PRW_1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>100PRW_2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>RDH</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>当前</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
